shell.go

```diff
@@ -20,11 +22,42 @@ import (
 	tar "github.com/whyrusleeping/tar-utils"
 )
 
+const (
+	DefaultPathName = ".ipfs"
+	DefaultApiFile  = "api"
```

Comment:
This isn't really a default since it's not overridable :) But definitely something to have in a const.